### ISSUE TITLE
Typing the text 'jk' in rapid succession throws me out of the INSERT mode

### ISSUE THREAD

**Issue by run-stop** [AstroNvim/AstroNvim#1324]
### Checklist

- [X] Using a stable version of Neovim (i.e. not neovim nightly)
- [X] `:PackerSync`
- [X] `:AstroUpdate`
- [X] Restarted AstroNvim

### AstroNvim version

v2.6.5

### Neovim version (>= 0.8)

0.8

### Operating system/version

Ubuntu 20.04

### Describe the bug

While in the INSERT mode, entering the text 'jk' in rapid succession throws me out of the insert mode and deletes the 'j' so that I end up in the NORMAL mode. This is not happening with the standard VI or NVIM.

Can you please help me disabling this in the AstroNvim, as 'jk' in a rapid succession is very common pattern in my day to day usage?

### Steps to Reproduce

1. Start the new AtroNvim session (or use the existing one).
2. Go the the Insert mode (for example, by typing 'i' or 'o')
3. In rapid succession type 'ij' on the keyboard. 
4. The 'i' character gets deleted, and you end in an empty line in the Normal mode.

### Expected behavior

The editor should continue accepting 'i' followed by 'j' as in "ijijijijij".

**Comment by run-stop** [comment]
@gzzinx, thank you for the link... Not sure how to disable the 'better-escape' plugin. I've tried commenting out the plugin in '.config/nvim/lua/core/plugins.lua', but after the restart I get the same problem.

**Comment by gzzinx** [comment]
@run-stop Place this code in `~/.config/nvim/lua/user/init.lua`
```
return {
  plugins = {
    init = {
      ["max397574/better-escape.nvim"] = { disable = true },
    },
  },
}
```

and then run `nvim +PackerSync`

**Comment by mehalter** [comment]
@gzzinx thanks for the detailed answer and helping out! One thing I would add is to make sure @run-stop you don't edit anything in the core files of AstroNvim and you stay in the `user/` folder and follow what's described in the docs: https://astronvim.github.io/Configuration/manage_user_config

Editing the core files is unsupported and will break the ability to update. 

Any time you modify `plugins.init` table you will need to restart and run `:PackerSync`

**Comment by run-stop** [comment]
Thank you All for your help. As I made some mess in the configuration, I had to remove all traces of the nvim configuration (~/.local/share/nvim and ~/.config/nvim). Then I reinstalled AstroNvim, created the ~/.config/nvim/lua/user folder and as @gzzinx suggested, saved the init.lua file. Then I did the nvim +PackerSync, and 'jk' is gone. Everything works as expected.

Thank you for your help and the lesson in proper nvim plugin management!
